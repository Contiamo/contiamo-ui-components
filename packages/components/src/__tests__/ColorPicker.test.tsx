import * as React from "react"
import { render } from "enzyme"

import wrapDefaultTheme from "../utils/wrap-default-theme"
import { ColorPicker as ThemelessColorPicker } from "../index"

const ColorPicker = wrapDefaultTheme(ThemelessColorPicker)

describe("ColorPicker", () => {
  it("Should initialize properly", () => {
    expect(render(<ColorPicker />)).toMatchSnapshot()
  })
})
