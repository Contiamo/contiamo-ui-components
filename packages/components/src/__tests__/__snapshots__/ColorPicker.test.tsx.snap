// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ColorPicker Should initialize properly 1`] = `
<div
  class="css-ih7tap"
>
  <button
    class="css-zzm73f"
  >
    #03f
  </button>
</div>
`;
